### ISSUE TITLE
Unable to run 0.12.0 (ENOENT node-inspector/tools/protocol.json)

### ISSUE THREAD

**Issue by baldwicc** [node-inspector/node-inspector#708]
I'm having problems with the latest 0.12.0 release - it looks like `/tools` is excluded from by `.npmignore`

````
$ npm --version
2.11.2

$ node --version
v0.12.5
````

Here's what happens when trying to run `node-inspector`:
````
$ node-inspector
Node Inspector v0.12.0
fs.js:500
  return binding.open(pathModule._makeLong(path), stringToFlags(flags), mode);
                 ^
Error: ENOENT, no such file or directory '/Users/baldwicc/.nvm/versions/node/v0.12.5/lib/node_modules/node-inspector/tools/protocol.json'
    at Error (native)
    at Object.fs.openSync (fs.js:500:18)
    at Object.fs.readFileSync (fs.js:352:15)
    at new ProtocolJson (/Users/baldwicc/.nvm/versions/node/v0.12.5/lib/node_modules/node-inspector/lib/plugins.js:123:36)
    at DebugServer.start (/Users/baldwicc/.nvm/versions/node/v0.12.5/lib/node_modules/node-inspector/lib/debug-server.js:89:24)
    at Object.<anonymous> (/Users/baldwicc/.nvm/versions/node/v0.12.5/lib/node_modules/node-inspector/bin/inspector.js:30:13)
    at Module._compile (module.js:460:26)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
````

Missing `tools/` where NPM installed to:
````
$ cd /Users/baldwicc/.nvm/versions/node/v0.12.5/lib/node_modules/node-inspector/ && ls
AUTHORS  CONTRIBUTING.md  ChangeLog.md  LICENSE  README.md  bin  docs  front-end  front-end-node  index.js  lib  node_modules  package.json
````

**Comment by ashelley** [comment]
I can confirm that this pull request in a fork fixes this issue for me:

https://github.com/baldwicc/node-inspector/pull/1

without this I was unable to run node-inspector after freshly installing from npm on windows 8.1 node version 0.12.7 node-inspector 0.12.0

**Comment by 3y3** [comment]
FIxed in 0.12.1
